Map updates

darkdrgn2k opened the pull request:
Ref issue  #218 
Corrected errors in dialog boxes overlaying
Added contact textbox and field
Included additional parameters in JSON

Added disclaimer in form of !

> Our networks is currently still under construction. If you are interested in helping build out the network, live in Toronto, and/or are willing to host a wireless directional antenna please get in touch with us. We do not have the infrastructure to provide "free Internet" at this time. 

![image](https://user-images.githubusercontent.com/29005789/54459451-17d62c80-473d-11e9-9bbf-1709f9610e02.png)

darkdrgn2k committed:
Corrected errors in dialog boxes
Added contact textbox
Included additional paramaters in JSON

darkdrgn2k committed:
Added under construction

darkdrgn2k committed:
Corrected spelling/syntax

darkdrgn2k committed:
Added disclaimer and corrected some codeclimate stuff

Shrinks99 commented:
Screenshot looks good at first glance, I would like to make a bunch of CSS changes and will hopefully get to that next week!

Shrinks99 committed:
Merge branch 'master' into map-update

Shrinks99 committed:
Unified button shadow & InfoDialog changes

Makes the ! button larger and more in-line with the other buttons on the page.  The wording of the infobox has been slightly adjusted

Changes all buttons sitewide to a 1px shadow blur, all the buttons on the map now use the same blur / shadow levels.

Map search now reads "search for location" in order to further the distinction between the add a node workflow (which needs to be changed in the future but that's a future thing).

Changes the sliders and removes the dumb labels.... I hate those labels :P  They're also snappier now.

Shrinks99 commented:
Changes made, good to go I think!
